test/box-luatest/cross_engine_tx_test.lua

```diff
@@ -0,0 +1,265 @@
+local t = require('luatest')
+
+local server = require('luatest.server')
+
+local g = t.group('cross_engine_tx', {
+    {mvcc = true,  engine1 = 'memtx', engine2 = 'vinyl'},
+    {mvcc = true,  engine1 = 'vinyl', engine2 = 'memtx'},
+    {mvcc = false, engine1 = 'memtx', engine2 = 'vinyl'},
+    {mvcc = false, engine1 = 'vinyl', engine2 = 'memtx'},
+})
+
+g.before_all(function(cg)
```

Comment:
Forgot to add `after_all` callback to drop the test server. Will fix on the next push.

Reply:
Fixed.